pkg/cmd/codespace/ssh.go

```diff
@@ -45,6 +87,8 @@ func newSSHCmd(app *App) *cobra.Command {
 	sshCmd.Flags().StringVarP(&opts.codespace, "codespace", "c", "", "Name of the codespace")
 	sshCmd.Flags().BoolVarP(&opts.debug, "debug", "d", false, "Log debug data to a file")
 	sshCmd.Flags().StringVarP(&opts.debugFile, "debug-file", "", "", "Path of the file log to")
+	sshCmd.Flags().BoolVarP(&opts.stdio, "stdio", "", false, "Proxy sshd connection to stdio")
```

Comment:
It's not something that makes sense to use anywhere other than an openssh `ProxyCommand` directive, so yeah, hiding it would make sense. It's possible it'll be useful to implementors of similar integration for other, non-openssh clients, but anyone doing that should be able to find `--stdio` in our generated configs.

Reply:
Hidden: https://github.com/cli/cli/pull/4915/commits/dc14ae48d0c715f1fee2bd78b57497c54afd570d